WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -80,7 +80,7 @@
       <c:PreviewItem Icon="{StaticResource entities_regular}"
                      Label="Labels"
                      TextValue="{Binding Labels}">
-        <c:PrivacyContentControl NumberOfPrivacyChars="20">
+        <c:PrivacyContentControl NumberOfPrivacyChars="20" MinWidth="180" VerticalContentAlignment="Center">
```

Comment:
The MinWidth breaks the position of the copy button. The bug you tried to fix here persists [elsewhere too](https://github.com/zkSNACKs/WalletWasabi/pull/9570#issuecomment-1320889767) on master. I suggest removing the minwidth quick fix and solving it everywhere in a separate PR.

![image](https://user-images.githubusercontent.com/16364053/202904877-3f385115-6b1b-4847-a14e-5a13a483ec0c.png)